appointment/upload-documents/uploadDocuments.php

```diff
@@ -73,21 +73,144 @@ class="submit dcf-btn dcf-btn-primary"
+					<li><b>Validating taxpayer's authentication (Reviewing photo identification & Social Security Cards/ITINS:</b> The site administrator will review the Client's uploaded documents ensuring receipt of signed Forms 14446 and 13614. When all documents are ready for review the Client will check the ready for tax preparation box on the secured link provided to them. The site coordinator will assign the Client to a certified return preparer who will then call the Client to review Form 13614.</li>
+					<li><b>Performing the interview with the taxpayer(s):</b> The Return Preparer will contact the Client by phone to discuss Form 13614 and review all photo identification, social security cards/ITIN letter and income documents that have been uploaded.</li>
+					<li><b>Preparing the tax return:</b> An IRS tax law certified return preparer will be assigned to the Client by the Site Coordinator. The Site Coordinator will assign a Client to a Return Prepare based upon the certification level of the Return Preparer.</li>
+					<li><b>Performing the quality review:</b> The Site Coordinator will review the Client's return.</li>
+					<li><b>Sharing the completed return:</b> The quality reviewed tax return will be uploaded and sent via secured link to the Client. The Return Preparer will call the Client to review the finished return, making any needed edits to the return, and asking the Client to sign Form 8879.</li>
+					<li><b>Signing the return:</b> The Client will upload on a secure link a signed and dated copy of their Form 8879.</li>
+					<li><b>E-filing the tax return:</b> The Site Coordinator will review and e-file the tax return.</li>
+				</ol>
+				
+				<p class="dcf-txt-h5 dcf-bold">Part III: Taxpayer Consents:</p>
+				<form class="cmxform dcf-form" id="vitaConsentForm" name="consentForm" ng-submit="submitConsent()" autocomplete="off" novalidate>
+					<p class="dcf-bold">Request to Review your Tax Return for Accuracy:</p>	
+					<p>To ensure you are receiving quality services and an accurately prepared tax return at the volunteer site,
+						IRS employees randomly select free tax preparation sites for review. If errors are identified, the site
+						will make the necessary corrections. IRS does not keep any personal information from your reviewed tax
+						return and this allows them to rate our VITA/TCE return preparation programs for accurately prepared tax
+						returns. If you do not wish to have your return included as part of the review process, it will not affect
+						the services provided to you at this site. If the site preparing this return is selected, do you consent
+						to having your return reviewed for accuracy, by an IRS employee?
+					</p>
+					<li class="form-radio" id="consentReview">
```

Comment:
Hm, you're right--this does look kinda strange. Maybe add a label that says "The IRS may quality review my tax return prepared by Lincoln VITA"

![image](https://user-images.githubusercontent.com/9295744/108143420-58f3bb80-708d-11eb-8971-94aafe121e39.png)